src/client.ts

```diff
@@ -34,14 +39,15 @@ const defaultClientConfiguration = {
 export class Client {
   /** The {@link ClientConfiguration} */
   readonly #clientConfiguration: ClientConfiguration;
-  /** The underlying {@link AxiosInstance} client. */
-  readonly client: AxiosInstance;
+  /** The underlying {@link HTTPClient} client. */
+  readonly client: HTTPClient;
```

Comment:
We agreed to rename this `#httpClient`, to 1) distinguish from a "client" as in an instance of the `Client` class, and 2) to make it private.

Reply:
any reason we can't make that change now?

Reply:
Yea, I intended to resolve this now.

Reply:
done!